examples/shell/shell_common/cmd_otcli.cpp

```diff
@@ -46,8 +46,13 @@
 #endif
 static char sTxBuffer[SHELL_OTCLI_TX_BUFFER_SIZE];
 static constexpr uint16_t sTxLength = SHELL_OTCLI_TX_BUFFER_SIZE;
-#endif // !CHIP_DEVICE_CONFIG_THREAD_ENABLE_CLI)
+#endif // !CHIP_DEVICE_CONFIG_THREAD_ENABLE_CLI
+// Use the max line value from the OT config, currently 384 but allow
+// redefining it in Matter config
+#ifndef SHELL_OTCLI_MAX_LINE_LEN
```

Comment:
None of these SHELL_OTCLI constants seem to be defined or overriden in matter config. The shell example is an example so we do not expect this to be a standard ABI outside.

If we do want tunable valies in one place, please create a config header specific to this example that you include.

Constant-wise, using `k....` is a better naming that C-style defines.